src/templates/ucp.html

```diff
@@ -2,10 +2,10 @@
 
 {% block body %}
 <ul id='menu'>
-        <li class='tabhandle' id='tab1'>{{ 'Preferences'|trans }}</li>
-        <li class='tabhandle' id='tab2'>{{ 'Account'|trans }}</li>
-        <li class='tabhandle' id='tab3'>{{ 'Templates'|trans }}</li>
-        <li class='tabhandle' id='tab4'>{{ 'API keys'|trans }}</li>
+  <li class='tabhandle' id='tab2'>{{ 'Account'|trans }}</li>
```

Comment:
why is this inverted?

Reply:
Good question, must be by mistake. I fixed it.